src/electron/future/utils/path-helpers.ts

```diff
@@ -12,6 +12,8 @@ export const resourcesDirectory = isDevelopment
 	? path.join(process.cwd(), "resources")
 	: path.join(app.getPath("exe"), "..", "resources", "app.asar.unpacked", "resources");
 
+export const userDataDirectory = isDevelopment ? process.cwd() : app.getPath("userData");
```

Comment:
can we add a dedicated folder here which we can then ignore. This approach won't scale as it could poolute the root directory

Reply:
actually, userData is already handled. it uses either `captain` or `captain__development__`

![image](https://github.com/blib-la/captain/assets/1148334/90a09e0d-4777-45ff-9ebe-402a70fe9cb5)

Reply:
My problem here is, that I can't use `app.getPath`, as this is something that only exists in electron. For testing I want to run my e2e tests without electron, as this allows me to directly execute everything and don't interact with my code via IPC. So both of these folders are not reachable for my tests at all. 

This approach is also not poluting the root, as the folder is .gitignored, so it will only be relevant for a person that wants to test this. 

This solution was something I came up with after trying many other things and all of them failed. But I will try one last thing and mock the app.getPath folder for this specific test.

Reply:
It worked, I mocked `app.getPath` with `process.cwd()` for my specific test. I removed all the other changes to `path-helpers`.